homeassistant/components/senseme/entity.py

```diff
@@ -0,0 +1,54 @@
+"""The SenseME integration entities."""
+from __future__ import annotations
+
+from aiosenseme import SensemeDevice
+
+from homeassistant.core import callback
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import DeviceInfo, Entity
+
+
+class SensemeEntity(Entity):
+    """Base class for senseme entities."""
+
+    _attr_should_poll = False
+
+    def __init__(self, device: SensemeDevice, name: str) -> None:
+        """Initialize the entity."""
+        self._device = device
+        self._attr_name = name
+        self._attr_device_info = DeviceInfo(
+            connections={(dr.CONNECTION_NETWORK_MAC, self._device.mac)},
+            name=self._device.name,
+            manufacturer="Big Ass Fans",
+            model=self._device.model,
+            sw_version=self._device.fw_version,
+            suggested_area=self._device.room_name,
+        )
+
+    @property
+    def extra_state_attributes(self) -> dict:
+        """Get the current device state attributes."""
+        return {
+            "room_name": self._device.room_name,
+            "room_type": self._device.room_type,
```

Comment:
These attributes seem static. We normally don't want static info in state attributes.

Reply:
I'm not sure how useful room_type, and I think `room_name` has been superseded by suggested_area anyways so I'm going to drop them

Reply:
https://github.com/home-assistant/core/pull/63748